pom.xml

```diff
@@ -41,7 +41,8 @@
     <spring.version>4.1.5.RELEASE</spring.version>
     <fcrepo.version>4.4.0</fcrepo.version>
     <fcrepo-audit.version>4.4.0</fcrepo-audit.version>
-    <fcrepo-camel.version>4.4.0</fcrepo-camel.version>
+    <fcrepo-camel.version>4.4.1-SNAPSHOT</fcrepo-camel.version>
```

Comment:
Note: The camel dependency version is changed to a SNAPSHOT version here. If this is not desired, we can wait until fcrepo-camel 4.4.1 is released. And, update and merge the PR after the release.